src/pubsub.js

```diff
@@ -27,7 +27,11 @@ function get() {
 
 	var pubsub;
 
-	if (nconf.get('redis')) {
+	if (nconf.get('isCluster') === 'true') {
```

Comment:
shouldn't it be `nconf.get('isCluster') === 'false'`

Reply:
Whoops! Good catch!